### PR TITLE
Browse Shares: multi-select in the folder view

### PR DESCRIPTION
**Pull request by stefantalpalaru**
It was already possible to select multiple files in the right panel. With this PR, it's now possible to select multiple folders in the left one.

### CODE REVIEW

**Comment by mathiascode** [comment]
Thanks! I think this is something we can add. I'll review the changes when I have time.

**Comment by mathiascode** [comment]
The folder context menu is currently reused for the dropdown menu in the path bar, which can result in the menu items targeting the wrong folder when you change the selection.

I think the easiest path forward would be to:

- Unset the active folder when multiple folders are selected, which in turn hides the path bar and dropdown menu
- Rename `self.selected_folder_path` to `self.active_folder_path` or `self.visible_folder_path`, and only use it in `set_selected_folder()`
- In all other cases, replace `self.selected_folder_path` with `self.folder_tree_view.get_selected_rows()`, like you've done in some cases already

File: pynicotine/gtkgui/userbrowse.py [hunk 1]
```diff
@@ -252,7 +252,7 @@ def __init__(self, userbrowses, user):
         # Setup folder_tree_view
         self.folder_tree_view = TreeView(
             self.window, parent=self.folder_tree_container, has_tree=True, always_select=True,
```
Comment (mathiascode) [inline pynicotine/gtkgui/userbrowse.py:254]:
```suggestion
            self.window, parent=self.folder_tree_container, has_tree=True,
```

Comment (stefantalpalaru) [reply pynicotine/gtkgui/userbrowse.py]: Done.

**Comment by stefantalpalaru** [comment]
Variable renamed as suggested. The path bar and file list view are now cleared when multiple folders are selected.

I'm not aware of any other cases where multiple folder selection needs to be handled.

**Pull request merged**